forms/PopoverField.php

```diff
@@ -47,19 +54,33 @@ public function setPopoverTitle($popoverTitle)
 		return $this;
 	}
 
+	/**
+	 * Get popup placement
+	 *
+	 * @return string
+	 */
+	public function getPlacement()
+	{
+		return $this->placement;
+	}
+
+	public function setPlacement($placement)
+	{
+		$valid = ['top', 'right', 'bottom', 'left'];
+
+		if (in_array($placement, $valid)) {
```

Comment:
That's a pretty nasty way to hide a fault parameter from devs (silently ignore) - I've made it raise an exception.